sample-parent/sample-features/fixed-schema/src/main/java/org/talend/sdk/component/sample/feature/fixedschema/processor/FixedSchemaProcessor.java

```diff
@@ -0,0 +1,82 @@
+/**
+ * Copyright (C) 2006-2025 Talend Inc. - www.talend.com
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.talend.sdk.component.sample.feature.fixedschema.processor;
+
+import java.io.Serializable;
+
+import org.talend.sdk.component.api.component.Icon;
+import org.talend.sdk.component.api.component.Version;
+import org.talend.sdk.component.api.meta.Documentation;
+import org.talend.sdk.component.api.processor.ElementListener;
+import org.talend.sdk.component.api.processor.Input;
+import org.talend.sdk.component.api.processor.Output;
+import org.talend.sdk.component.api.processor.OutputEmitter;
+import org.talend.sdk.component.api.processor.Processor;
+import org.talend.sdk.component.api.record.Record;
+import org.talend.sdk.component.api.service.record.RecordBuilderFactory;
+import org.talend.sdk.component.api.service.schema.FixedSchema;
+import org.talend.sdk.component.sample.feature.fixedschema.config.Config;
+import org.talend.sdk.component.sample.feature.fixedschema.service.UIService;
+
+@Version
+@Processor(name = "Processor")
```

Comment:
If we plan to make lot of sample connectors maybe have a unic name for all?
```suggestion
@Processor(name = "ProcessorFixedSchema")
```

Reply:
As you said If the family is set as family = "sampleFixedSchema" and the name as name = "Processor", this results in the component being named tSampleFixedSchemaProcessor.

My full proposal (linked to the familly comment) suggests that instead of this, we could:

- Use family = "featureSample" or just "Sample"
- Use name = "ProcessorFixedSchema"

This would result in either:

- tFeatureSampleProcessorFixedSchema or
- tSampleFixedSchemaProcessor.

This way, the result would remain the same, but there would only be one family. This means we would have one distinct family icon and individual connector icons for the components, which seems more logical.

If the family is always dependent on the connector, it's not really a family anymore but more like a pluginName, isn't it?

Reply:
I prefer to have 1 family by feature to not install too much connectors in the studio and to prevent connectors id conflict.

Reply:
And also to avoid risk of duplicate connectors name as discussed together. I close the topic.